CONTRIBUTING.MD

```diff
@@ -0,0 +1,64 @@
+Lyra Contributing Guidelines
+=======================
+
+New to Lyra? Check out "First PR" tag in the issues. These are issues have a lower level of effort for people looking to get their hands dirty in the Lyra codebase.
+
+## Guideline Contents
+
+There are a lot of ways to get involved and help out:
+
+- [Reporting an issue](#reporting-issues)
+- [Requesting features](#requesting-features)
+- [Submitting Pullrequests](#submitting-prs)
+- [Writing Documentation & tutorials](#documentation)
+- [Contributing Design Assets](#contributing-design)
+
+<a name="reporting-issues"></a>
+## Reporting Issues
+
+When creating a new issue for a bug please include the following:
+- **the browser you discovered the bug in**
+- **a description of the problem**
+- **a description of what you expected to happen**
+
+If it's applicable, include a gif of the interaction. You can capture your screen as a gif using [LiceCap](http://www.cockos.com/licecap/).*
```

Comment:
I'd say gif or image. Sometimes the issue can be seen statically too, so we don't need to trouble folks for gifs too.